packages/truffle-debugger/lib/solidity/selectors/index.js

```diff
@@ -269,6 +266,71 @@ let solidity = createSelectorTree({
       }
     ),
 
+    /*
+     * solidity.current.functionsByPc
```

Comment:
This is cool!

Naming suggestion: I feel like we should spell out `programCounter`. I think it'll read better: `solidity.current.functionsByProgramCounter`, and so on.

Reply:
Makes sense.  Will go change that.